src/QMCWaveFunctions/Fermion/DelayedUpdate.h

```diff
@@ -106,7 +106,7 @@ class DelayedUpdate
     }
     const T cone(1);
     const T czero(0);
-    const int norb     = Ainv.rows();
+    const int norb     = Ainv.cols();
```

Comment:
Ainv is the inverse transpose of A. So norb = Ainv.rows() = A.rows().
@spinedaflores did you get any trouble if leaving this unchanged?

Reply:
You're right. It seems that we don't need that change. 

On my first attempt of doing the pull request I found that norb was being incorrectly set to number of electrons (when more than the minimal number of orbitals are being read in). I must have changed this when I was trying to figure out why earlier. But it looks like we don't need this change

Reply:
@spinedaflores Please fix these changes.